static/css/partials/_partials.scss

```diff
@@ -4,13 +4,11 @@
 @import "base";
 @import "layout";
 @import "animations";
+@import "theme";
```

Comment:
This order change is causing  .button.active background-color inside _buttons.scss to now override  .button.transparent in theme, resulting in edit tools pen, undo and redo button to have a background-color

Reply:
Debugging some more I don't think "active on button" logic as intended is even functional for edit annotation tools https://github.com/mozilla-services/screenshots/blob/master/server/src/pages/shot/editor.js#L182
and should be fixed and could be a separate issue.